src/common.rs

```diff
@@ -1267,7 +1267,7 @@ pub fn save_access_key_to_keychain(
     keyring::Entry::new(&service_name, &format!("{}:{}", account_id, public_key_str))
         .wrap_err("Failed to open keychain")?
         .set_password(key_pair_properties_buf)
-        .wrap_err("Failed to save password to keychain")?;
+        .wrap_err("Failed to save password to keychain. You may need to install secure keychain package by following  this instructions: https://github.com/jaraco/keyring#using-keyring-on-headless-linux-systems")?;
```

Comment:
- Is it okay that we have a double-space between words (`following  this`)?
- Last sentence isn't grammatically correct, it should be `these instructions` or `this instruction` (I guess first one is better)
- `the secure keychain` looked correct in a previous version. Why an article was removed in the last commit?